CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/templates/SimpleEditPlugin.java

```diff
@@ -116,34 +107,17 @@ public final void run(final PluginGraphs graphs, final PluginInteraction interac
                         if (!"Editing...".equals(interaction.getCurrentMessage())) {
                             inControlOfProgress = false;
                         }
-                    } catch (InterruptedException e) {
-                        cancelled = true;
-                        interaction.notify(PluginNotificationLevel.INFO, "Action cancelled: " + graphs.getGraph() + SeparatorConstants.COLON + " " + getName());
-                        throw e;
-                    } catch (PluginException e) {
-                        final String msg = Bundle.MSG_Edit_Failed(graph, getName());
-                        interaction.notify(PluginNotificationLevel.ERROR, msg + SeparatorConstants.NEWLINE + e.getMessage());
-                        cancelled = true;
-                        LOGGER.log(Level.WARNING, msg, e);
-                        throw e;
-                    } catch (Exception ex) {
-                        final String msg0 = String.format("Unexpected non-plugin exception caught in %s.run()", SimpleEditPlugin.class.getName());
-                        final String msg = Bundle.MSG_Edit_Failed(graph, getName());
-                        interaction.notify(PluginNotificationLevel.ERROR, msg0 + SeparatorConstants.SEMICOLON + SeparatorConstants.NEWLINE + msg + SeparatorConstants.NEWLINE + ex.getMessage());
+                    } catch (final Exception ex) {
                         cancelled = true;
-                        LOGGER.log(Level.WARNING, ex, () -> msg0 + SeparatorConstants.SEMICOLON + " " + msg);
-                        throw new RuntimeException(ex);
```

Comment:
Deciding not to wrap the extension in a RuntimeException() - https://softwareengineering.stackexchange.com/questions/121328/is-it-good-practice-to-catch-a-checked-exception-and-throw-a-runtimeexception